examples/kg-chat/cypher_message.py

```diff
@@ -0,0 +1,41 @@
+CONSTRUCT_DEPENDENCY_GRAPH = """
```

Comment:
add comment on what this does.
Could this be generated by the LLM? (Or actually it probably WAS generated by ChatGPT4, but could we have this be part of the process, so we don't need to hardcode this? or is it better to keep this here to keep it deterministic?)

Reply:
It wasn't generated by ChatGPT4 and haven't tried it. I doubt ChatGPT can generate this because it's not aware of the intended schema (nodes labels and their relationships).
So hardcoding it is the safest way to make sure the graph database is constructed as intended.
On the other hand, constructing the graph database for CSV can be done using LLM is easier (working on it now) because all what you need to provide is the CSV header. Then LLM will infer the nodes, their labels, and relationships.